tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -176,6 +180,17 @@ protected virtual ITraceSampler GetSampler(ImmutableTracerSettings settings)
             return sampler;
         }
 
+        protected virtual ISpanSampler GetSpanSampler(ImmutableTracerSettings settings)
+        {
+            if (string.IsNullOrWhiteSpace(settings.SpanSamplingRules))
+            {
+                // TODO should this be null?
```

Comment:
Another way to do it is to mimic what had been done for TraceSampler. ie always build the sampler, and it will always return false if not needed. 
I'd say I have a slight preference for this later approach as there's no risk to forget a `?` somewhere and because we keep the same mechanism. But no strong feeling either way.

Reply:
And to add up to the confusion, the last scenario you keep it null and you use the same pattern for TraceSampler. If we can avoid different setup I'll be glad.

Reply:
I opted to initialize it with an empty rule which should just return false in 4bc1743de0f0c7315c5d0da28913ea5b60cbf776